release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -612,6 +612,8 @@ For more information see xref:../scalability_and_performance/cnf-talm-for-cluste
 
 [id="ocp-4-13-insights-operator"]
 === Insights Operator
+// https://issues.redhat.com/browse/OCPBUGS-6832
+The Insights Operator can now collect the `openshift_apps_deploymentconfigs_strategy_total` metric. This metric gathers deployment strategy information from a deployment's configuration.
```

Comment:
```suggestion
With this release, the Insights Operator collects the `openshift_apps_deploymentconfigs_strategy_total` metric. This metric gathers deployment strategy information from a deployment's configuration.
```

Reply:
I'm unsure of this as the "release" is already implied.